boards/arm/nrf52833dk_nrf52833_mdbt/nrf52833dk_nrf52833_mdbt.dts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022 Nordic Semiconductor ASA
+// SPDX-License-Identifier: Apache-2.0
+
+/dts-v1/;
+#include <nordic/nrf52833_qiaa.dtsi>
+
+/ {
+	model = "NRF52833DK_NRF52833_MDBT";
+	compatible = "raytac,nrf52833dk-nrf52833-mdbt";
+
```

Comment:
Please name this board appropriately, if the board is called e.g. `MDBT50Q` then the board name and folder name should be `mdbt50q`